docs/algorithm/algorithm.tex

```diff
@@ -0,0 +1,278 @@
+\documentclass{article}
+
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{graphicx}
+
+\DeclareMathOperator{\erf}{erf}
+\DeclareMathOperator{\Var}{Var}
+\DeclareMathOperator{\Cov}{Cov}
+
+\begin{document}
+
+\section{Global architecture}
+
+The clock control filter of NTPD-rs consists of 4 stages.
+
+First, on a per-time-source basis, a kalman filter is used to estimate the offset and frequency error as compared to that specific timesource.
```

Comment:
Nice intro, but this could use a graphic like the one on the whiteboard.

Reply:
Added graphic